packages/cordova/README.md

```diff
@@ -0,0 +1,12 @@
+# @padloc/cordova
```

Comment:
This file is mostly so everyone knows what's necessary to have pre-installed, rather than running commands and being told step by step.